### ISSUE TITLE
[Qt5] build failure - vcpkg freezes  during Qt5 build

### ISSUE THREAD

**Issue by nickanthony-dgl** [microsoft/vcpkg#25634]
**Host Environment**
- OS: Windows 10
- Compiler: MSVC 14.32

I have tried installing Qt5 in many different ways but it has never succeeded. I initially was trying to install it using manifest mode using a custom triplet but I have also tried in many times directly through in CMD.exe and using default triplets (x86 and x64)
e.g. `./vcpkg.exe install qt5:x64-windows`. I was initially working with a version of vcpkg from May, but I have now updated to a July 6th version.

The build stops progressing while building qt5-base but I also saw one time where qt5-base was successful and then qt5-tools ended up freezing. Interestingly this never occurs at the exact same part of the build, e.g. sometime the last line on the console is "Building x64-windows-dbg" other times it will be during the release build, or sometimes it will say "Installing ....".  I can see that there are 2 `cmake` processes running as well as `vcpkg` and `jom`, but none of them are utilizing even 1% of CPU.

**Failure logs**
I'm really not sure where to find the correct log files to help me figure out what is going on. Here are logs from buildtrees/qt5-base

build-x64-windows-dbg-err:
```
jom 1.1.3 - empower your cores

cl : Command line warning D9025 : overriding '/Fo./' with '/Foqlibraryinfo_final.obj'
WARNING: Failure to find: C:/vcpkg/buildtrees/qt5-base/x64-windows-wavetracer-dbg/src/corelib/qt5cored_metatypes.json
WARNING: Failure to find: C:/vcpkg/buildtrees/qt5-base/x64-windows-wavetracer-dbg/src/corelib/qt5core_metatypes.json
C:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\src\corelib\kernel\qobjectdefs.h(0): Note: No relevant classes found. No output generated.
WARNING: Failure to find: C:/vcpkg/buildtrees/qt5-base/x64-windows-wavetracer-dbg/src/gui/qt5guid_metatypes.json
WARNING: Failure to find: C:/vcpkg/buildtrees/qt5-base/x64-windows-wavetracer-dbg/src/gui/qt5gui_metatypes.json
C:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\src\network\access\qnetworkaccessdebugpipebackend_p.h(0): Note: No relevant classes found. No output generated.

```

end of build-x64-windows-dbg-out:
```
	cl -c -FI..\..\corelib\global\qt_pch.h -Yu..\..\corelib\global\qt_pch.h -Fp.pch\debug\Qt5FontDatabaseSupportd_pch.pch -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -utf-8 /wd4530 /wd4577 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 /Fd../../../lib/Qt5FontDatabaseSupportd.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_NO_LINKED_LIST -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_CAST_FROM_ASCII -DQT_USE_DIRECTWRITE2 -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_BUILD_FONTDATABASE_SUPPORT_LIB -DQT_BUILDING_QT -D_CRT_SECURE_NO_WARNINGS -D_USE_MATH_DEFINES -DQT_NO_CAST_TO_ASCII -DQT_ASCII_CAST_WARNINGS -DQT_MOC_COMPAT -DQT_USE_QSTRINGBUILDER -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_BEFORE=0x040800 -DQT_DEPRECATED_WARNINGS_SINCE=0x060000 -DQT_NO_EXCEPTIONS -DQT_GUI_LIB -DQT_CORE_LIB -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\src\platformsupport\fontdatabases -I. -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtFontDatabaseSupport -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtFontDatabaseSupport\5.15.5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtFontDatabaseSupport\5.15.5\QtFontDatabaseSupport -I..\..\..\include -I..\..\..\include\QtFontDatabaseSupport -Itmp -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtGui\5.15.5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtGui\5.15.5\QtGui -I..\..\..\include\QtGui\5.15.5 -I..\..\..\include\QtGui\5.15.5\QtGui -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtCore\5.15.5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtCore\5.15.5\QtCore -I..\..\..\include\QtCore\5.15.5 -I..\..\..\include\QtCore\5.15.5\QtCore -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtGui -I..\..\..\include\QtGui -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtCore -I..\..\..\include\QtCore -I.moc\debug -IC:\Users\NicholasAnthony\source\repos\wt_gui\out\build\MSVC-Debug\vcpkg_installed\x64-windows-wavetracer\include -IC:/Users/NicholasAnthony/source/repos/wt_gui/out/build/MSVC-Debug/vcpkg_installed/x64-windows-wavetracer/include/qt5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\mkspecs\win32-msvc -Fo.obj\debug\ @C:\Users\NICHOL~1\AppData\Local\Temp\qwindowsnativeimage.obj.24196.2282.jom
qwindowsnativeimage.cpp
	cl -c -FI..\..\corelib\global\qt_pch.h -Yu..\..\corelib\global\qt_pch.h -Fp.pch\debug\Qt5FontDatabaseSupportd_pch.pch -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -utf-8 /wd4530 /wd4577 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 /Fd../../../lib/Qt5FontDatabaseSupportd.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_NO_LINKED_LIST -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_CAST_FROM_ASCII -DQT_USE_DIRECTWRITE2 -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_BUILD_FONTDATABASE_SUPPORT_LIB -DQT_BUILDING_QT -D_CRT_SECURE_NO_WARNINGS -D_USE_MATH_DEFINES -DQT_NO_CAST_TO_ASCII -DQT_ASCII_CAST_WARNINGS -DQT_MOC_COMPAT -DQT_USE_QSTRINGBUILDER -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_BEFORE=0x040800 -DQT_DEPRECATED_WARNINGS_SINCE=0x060000 -DQT_NO_EXCEPTIONS -DQT_GUI_LIB -DQT_CORE_LIB -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\src\platformsupport\fontdatabases -I. -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtFontDatabaseSupport -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtFontDatabaseSupport\5.15.5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtFontDatabaseSupport\5.15.5\QtFontDatabaseSupport -I..\..\..\include -I..\..\..\include\QtFontDatabaseSupport -Itmp -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtGui\5.15.5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtGui\5.15.5\QtGui -I..\..\..\include\QtGui\5.15.5 -I..\..\..\include\QtGui\5.15.5\QtGui -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtCore\5.15.5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtCore\5.15.5\QtCore -I..\..\..\include\QtCore\5.15.5 -I..\..\..\include\QtCore\5.15.5\QtCore -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtGui -I..\..\..\include\QtGui -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtCore -I..\..\..\include\QtCore -I.moc\debug -IC:\Users\NicholasAnthony\source\repos\wt_gui\out\build\MSVC-Debug\vcpkg_installed\x64-windows-wavetracer\include -IC:/Users/NicholasAnthony/source/repos/wt_gui/out/build/MSVC-Debug/vcpkg_installed/x64-windows-wavetracer/include/qt5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\mkspecs\win32-msvc -Fo.obj\debug\ @C:\Users\NICHOL~1\AppData\Local\Temp\qwindowsfontdatabase_ft.obj.24196.2313.jom
qwindowsfontdatabase_ft.cpp
	cl -c -FI..\..\corelib\global\qt_pch.h -Yu..\..\corelib\global\qt_pch.h -Fp.pch\debug\Qt5FontDatabaseSupportd_pch.pch -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -utf-8 /wd4530 /wd4577 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 /Fd../../../lib/Qt5FontDatabaseSupportd.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_NO_LINKED_LIST -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_CAST_FROM_ASCII -DQT_USE_DIRECTWRITE2 -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_BUILD_FONTDATABASE_SUPPORT_LIB -DQT_BUILDING_QT -D_CRT_SECURE_NO_WARNINGS -D_USE_MATH_DEFINES -DQT_NO_CAST_TO_ASCII -DQT_ASCII_CAST_WARNINGS -DQT_MOC_COMPAT -DQT_USE_QSTRINGBUILDER -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_BEFORE=0x040800 -DQT_DEPRECATED_WARNINGS_SINCE=0x060000 -DQT_NO_EXCEPTIONS -DQT_GUI_LIB -DQT_CORE_LIB -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\src\platformsupport\fontdatabases -I. -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtFontDatabaseSupport -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtFontDatabaseSupport\5.15.5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtFontDatabaseSupport\5.15.5\QtFontDatabaseSupport -I..\..\..\include -I..\..\..\include\QtFontDatabaseSupport -Itmp -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtGui\5.15.5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtGui\5.15.5\QtGui -I..\..\..\include\QtGui\5.15.5 -I..\..\..\include\QtGui\5.15.5\QtGui -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtCore\5.15.5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtCore\5.15.5\QtCore -I..\..\..\include\QtCore\5.15.5 -I..\..\..\include\QtCore\5.15.5\QtCore -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtGui -I..\..\..\include\QtGui -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\include\QtCore -I..\..\..\include\QtCore -I.moc\debug -IC:\Users\NicholasAnthony\source\repos\wt_gui\out\build\MSVC-Debug\vcpkg_installed\x64-windows-wavetracer\include -IC:/Users/NicholasAnthony/source/repos/wt_gui/out/build/MSVC-Debug/vcpkg_installed/x64-windows-wavetracer/include/qt5 -IC:\vcpkg\buildtrees\qt5-base\src\5.15.5-451bf817fc.clean\mkspecs\win32-msvc -Fo.obj\debug\ @C:\Users\NICHOL~1\AppData\Local\Temp\qwindowsfontenginedirectwrite.obj.24196.2329.jom
qwindowsfontenginedirectwrite.cpp
	lib /NOLOGO  /OUT:..\..\..\lib\Qt5FontDatabaseSupportd.lib @C:\Users\NICHOL~1\AppData\Local\Temp\Qt5FontDatabaseSupportd.lib.24196.5204.jom
```

**Additional context**
I know that this is very likely not enough information for someone to be able to help me. I'm hoping that someone can help point me to the correct log files to get more information about what is going wrong.

**Comment by Neumann-A** [comment]
modify your triplet to contain `set(VCPKG_QMAKE_USE_NMAKE ON)`. It is probably an issue with jom.

**Comment by nickanthony-dgl** [comment]
Thanks @Neumann-A , I will give that a try. I finally got everything to finish building after retrying ~20 times but I have noticed another potential issue that may be related.

It seems like vcpkg is using two triplets for some reason. It uses my custom triplet but it also installs many libraries using the default `x64-windows` triplet.

My top-level CMake file starts like this:
```cmake
cmake_minimum_required (VERSION 3.20)

if(DEFINED ENV{VCPKG_ROOT} AND NOT DEFINED CMAKE_TOOLCHAIN_FILE)
    set(CMAKE_TOOLCHAIN_FILE "$ENV{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake" CACHE STRING "")  # This must happen before `project`
    set(VCPKG_OVERLAY_TRIPLETS "${CMAKE_CURRENT_SOURCE_DIR}/custom-triplets/" CACHE STRING "")  # Use our locally stored custom triplet.
    set(VCPKG_TARGET_TRIPLET x64-windows-myproject)   # allows us to statically most libraries, but not Qt, CRT.
endif()
```

However when configuration starts I get this on the console:
```
1> [CMake] -- Running vcpkg install
1> [CMake] Detecting compiler hash for triplet "x64-windows"...
1> [CMake] Detecting compiler hash for triplet "x64-windows-myproject"...
1> [CMake] The following packages will be built and installed:
1> [CMake]   * assimp[core]:x64-windows-myproject-> 5.2.4
1> [CMake]   * brotli[core]:x64-windows-myproject-> 1.0.9#2
1> [CMake]   * brotli[core]:x64-windows -> 1.0.9#2
1> [CMake]   * bzip2[core]:x64-windows-myproject-> 1.0.8#2
1> [CMake]   * bzip2[core]:x64-windows -> 1.0.8#2
```

When configuration completes I can see in my build tree that VCPKG really has installed two copied of all of these libraries.


My custom triplet is:
```cmake
# Based off of x64-windows-static-md except specifying dynamic linkage for some libraries due to licensing constraints.
set(VCPKG_TARGET_ARCHITECTURE x64)
set(VCPKG_CRT_LINKAGE dynamic)
set(VCPKG_LIBRARY_LINKAGE static) # Default to static linkage if not specified otherwise

#Dynamic link LGPL libraries: Qt, (TODO are there more?)
set(VCPKG_QMAKE_USE_NMAKE ON) # This should help avoid a deadlock while building Qt: See vcpkg github #4692
if(PORT MATCHES "qt")
    set(VCPKG_LIBRARY_LINKAGE dynamic)
endif()

```

My vcpkg.json manifest is:
```json
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
  "name": "myproject",
  "version-semver": "0.0.1",
  "dependencies": [
    "protobuf",
    "eigen3",
    "gtest",
    {
      "name": "qt5",
      "default-features": false,
      "features": ["3d", "tools"]
    },
    {
      "name": "highfive",
      "default-features": false,
      "features": [ "eigen3" ]
    }
  ],
  "supports": "windows"
}
```

**Comment by Neumann-A** [comment]
> is using two triplets 

pass `--host-triplet=<yourtriplet>`

**Comment by nickanthony-dgl** [comment]
Thank you!

**Comment by mi-robin** [comment]
@nickanthony-dgl 
I'm facing the same problem, but in my case I'd love to just vcpkg install qt5:x64-windows. I've tried
- adding NMAKE instruction to builtin `x64-windows.cmake`
```
set(VCPKG_TARGET_ARCHITECTURE x64)
set(VCPKG_CRT_LINKAGE dynamic)
set(VCPKG_LIBRARY_LINKAGE dynamic)
# qt5
set(VCPKG_QMAKE_USE_NMAKE ON)
```
- adding host triplet to command line: `vcpkg install qt5:x64-windows --host-triplet=x64-windows`. Yes, I'm on standard Windows10 machine
- trying random triplets supported by vcpkg

None worked. I'm still stuck at 
```
-- Using cached mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.4.6-1-x86_64.pkg.tar.zst.
-- Using msys root at D:/desktop/_dev/vcpkg/downloads/tools/msys2/6f3fa1a12ef85a6f
-- Configuring x64-windows-dbg
```

Is there a complete procedure for getting qt5 installed on x64-windows?
Thanks a lot!

**Comment by mi-robin** [comment]
Some progress:
After moving `set(VCPKG_QMAKE_USE_NMAKE ON)` up so that the x64-windows.cmake now looks like
```
# qt5
set(VCPKG_QMAKE_USE_NMAKE ON)
set(VCPKG_TARGET_ARCHITECTURE x64)
set(VCPKG_CRT_LINKAGE dynamic)
set(VCPKG_LIBRARY_LINKAGE dynamic)
```
The build no longer gets stuck.
But now I hit an error during building `qt5-activeqt`
```
-- Configuring x64-windows-rel
-- Configuring x64-windows-rel done
-- Configuring x64-windows-dbg
-- Configuring x64-windows-dbg done
-- Package build-x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: "C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/nmake.exe" -j 21
    Working Directory: D:/desktop/_dev/vcpkg/buildtrees/qt5-activeqt/x64-windows-dbg
    See logs for more information:
      D:\desktop\_dev\vcpkg\buildtrees\qt5-activeqt\package-build-x64-windows-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_qmake.cmake:3 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_build_qmake.cmake:76 (z_run_jom_build)
  installed/x64-windows/share/qt5/qt_build_submodule.cmake:11 (vcpkg_build_qmake)
  installed/x64-windows/share/qt5/qt_submodule_installation.cmake:9 (qt_build_submodule)
  ports/qt5-activeqt/portfile.cmake:2 (qt_submodule_installation)
  scripts/ports.cmake:147 (include)


error: building qt5-activeqt:x64-windows failed with: BUILD_FAILED
Elapsed time to handle qt5-activeqt:x64-windows: 3.7 s
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+qt5-activeqt
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?title=[qt5-activeqt]+Build+error&body=Copy+issue+body+from+D%3A%2Fdesktop%2F_dev%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md
```
I've git-pulled the latest vcpkg from upstream.

**Comment by mi-robin** [comment]
Reported as #33817

**Comment by AlienLcb** [comment]
> Some progress: After moving `set(VCPKG_QMAKE_USE_NMAKE ON)` up so that the x64-windows.cmake now looks like
> 
> ```
> # qt5
> set(VCPKG_QMAKE_USE_NMAKE ON)
> set(VCPKG_TARGET_ARCHITECTURE x64)
> set(VCPKG_CRT_LINKAGE dynamic)
> set(VCPKG_LIBRARY_LINKAGE dynamic)
> ```
> 
> The build no longer gets stuck. But now I hit an error during building `qt5-activeqt`
> 
> ```
> -- Configuring x64-windows-rel
> -- Configuring x64-windows-rel done
> -- Configuring x64-windows-dbg
> -- Configuring x64-windows-dbg done
> -- Package build-x64-windows-dbg
> CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
>     Command failed: "C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/nmake.exe" -j 21
>     Working Directory: D:/desktop/_dev/vcpkg/buildtrees/qt5-activeqt/x64-windows-dbg
>     See logs for more information:
>       D:\desktop\_dev\vcpkg\buildtrees\qt5-activeqt\package-build-x64-windows-dbg-err.log
> 
> Call Stack (most recent call first):
>   scripts/cmake/vcpkg_build_qmake.cmake:3 (vcpkg_execute_build_process)
>   scripts/cmake/vcpkg_build_qmake.cmake:76 (z_run_jom_build)
>   installed/x64-windows/share/qt5/qt_build_submodule.cmake:11 (vcpkg_build_qmake)
>   installed/x64-windows/share/qt5/qt_submodule_installation.cmake:9 (qt_build_submodule)
>   ports/qt5-activeqt/portfile.cmake:2 (qt_submodule_installation)
>   scripts/ports.cmake:147 (include)
> 
> 
> error: building qt5-activeqt:x64-windows failed with: BUILD_FAILED
> Elapsed time to handle qt5-activeqt:x64-windows: 3.7 s
> Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
> Then check for known issues at:
>     https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+qt5-activeqt
> You can submit a new issue at:
>     https://github.com/microsoft/vcpkg/issues/new?title=[qt5-activeqt]+Build+error&body=Copy+issue+body+from+D%3A%2Fdesktop%2F_dev%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md
> ```
> 
> I've git-pulled the latest vcpkg from upstream.

Pls did you fix it？I'm having the same issue
